control-plane/api/v1alpha1/peeringacceptor_webhook_test.go

```diff
@@ -0,0 +1,159 @@
+package v1alpha1
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+
+	logrtest "github.com/go-logr/logr/testing"
+	"github.com/stretchr/testify/require"
+	admissionv1 "k8s.io/api/admission/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
```

Comment:
Tests webhook level validations (comparing to existing peering acceptor resources)